docs/examples/Ch-ExamplesAddingModbusDevice.rst

```diff
@@ -2,459 +2,273 @@
-
-**Importing APIs**
-
-In this section you will be using the program Postman to interact with EdgeX. You will also need to have the file “core-metadata.raml” available to load into the Postman application.  The file “core-metadata.raml” can be found here: “edgex/core-metadata…./src/test/resources/raml/core-metadata.raml”
-
-**Viewing available APIs**
+Modbus Device (Simulator)
+=========================
 
-* Open Postman
-* Click on the Import button
-* Add the file to the import dialog box - the application will take a about 30 seconds to digest the file you added.
-* If a list of API commands do not show up on the left hand side of the application then click on the “Collections” tab to the right of the “History” tab.
+http://modbuspal.sourceforge.net/
 
-**Create an addressable**
+To simulate sensors, such as temperature and humidity, do the following:
 
-* In the “Collections” tab, select the option “POST /addressable action
-* Open the body tab
-* Modify its contents
+1. Add two mock devices:
 
-  * name: mbus-rth-lcd-address
-  * protocol: OTHER (needs to be in ALL CAPS)
-  * address: /dev/ttyS5,19200,8,1,0
+.. image:: addmockdevices.png
+    :scale: 50%
+    :alt: Add Mock Devices
 
-    * (actual path to device, baud rate, data bits, stop bits, parity)
-    * (parity - none = 0, odd = 1, even = 2)
+2. Add registers according to the device manual:
 
-  * port: leave unchanged (since using OTHER, it is ignored)
-  * path: 254 (this is where you enter the Modbus Slave ID)
-  * publisher, user, password, topic - do not need to be modified
+    .. image:: addregisters.png
+        :scale: 50%
+        :alt: Add Registers
 
-* Press the “Send” button when you are finished
-* Note the addressable id
+3. Add the ModbusPal support value auto-generator, which can bind to registers:
 
-**Upload the profile**
+    .. image:: addvaluegen.png
+        :scale: 50%
+        :alt: Add Device Value Generators
 
-* In the “Collections” tab select the option “POST /deviceprofile/uploadfile
-* Open the body tab
+    .. image:: bindvalue.png
+        :scale: 50%
+        :alt: Bind Value Generator
 
-  * Under “Key”, look for the drop down menu for “text”. Be sure to write “file” in the open box.
-  * Under “Value” click  “Choose Files”, locate your profile file.
 
-* Press Upload
-* Press the “Send” button when you are finished
-* Note the profile id
+Set Up Before Starting Services
+===============================
 
-**Post the device**
+The following sections describe how to complete the set up before starting the services. If you prefer to start the services and then add the device, see `Set Up After Starting Services`_
 
-* In the “Collections” tab select the option “POST /device
-* Click on the “Body” tab
-* Modify its contents
+Set Up Device Profile
+---------------------
 
-  * There are three components that are required to be modified. They are:
+The DeviceProfile defines the device's values and operation method, which can be Read or Write. 
```

Comment:
Again, please make the Device Profile downloadable or link to it on Github

Reply:
Updated

Reply:
You need to add the file mqtt.test.device.profile.yml to this directory for the download link to work